core/src/main/scala/org/broadinstitute/dsde/rawls/monitor/CloneWorkspaceFileTransferMonitor.scala

```diff
@@ -39,7 +39,13 @@ class CloneWorkspaceFileTransferMonitorActor(val dataSource: SlickDataSource, va
         dataAccess.cloneWorkspaceFileTransferQuery.listPendingTransfers()
       }
       _ <- pendingTransfers.toList.traverse { pendingTransfer =>
-        IO.fromFuture(IO(copyBucketFiles(pendingTransfer)))
+        IO.fromFuture(IO(copyBucketFiles(pendingTransfer))).attempt.map {
+          case Left(e) => {
+            logger.warn(s"Failed to copy files from ${pendingTransfer.sourceWorkspaceBucketName} to ${pendingTransfer.destWorkspaceBucketName}", e)
+            List.empty
```

Comment:
so if I'm understanding this, this catches the error (which is ok because this is an async call) so that it doesn't affect other file transfers. 
If the file transfer fails all retries, would users know other than noticing that files are missing?

Reply:
For the future would be good to figure out an error recovery process. E.g. maybe we try to copy N times, and report to the user somehow if we still couldn't copy the files. As written now it will continually retry forever and generate a lot of errors in Rawls log.

Reply:
Agreed, filed a ticket on the CI board https://broadworkbench.atlassian.net/browse/CA-1542. I am doing a bit more digging into the offending row in prod that's affecting users, but it does seem like we might want to delete that row manually